### DISCUSSION TITLE
zh/tutorial/

### DISCUSSION THREAD

**Discussion by giscus** [ZLMediaKit/docs#11] [Announcements]
# zh/tutorial/

本教程将指导您完成编译和运行 ZLMediaKit。 构建环境 新手提示 如果你是位新手，强烈建议使用 Ubuntu16 以上版本编译 ZLMediaKit，macOS 是次选推荐平台。不推荐使用 CentOS6.x 或 windows 构建与使用。 vcpkg ZLMediaKit 已上架 vcpkg，便捷安装请参考 vcpkg 安装 ZLMedia...

https://docs.zlmediakit.com/zh/tutorial/

**Comment by wenlf** [comment]
great job.

**Comment by rongzedong** [comment]
1. 配套的 vcpkg 最后更新日期是 2024年，比较久，不建议使用。
2. 默认的证书已经过期，建议考虑给更新一下。

感谢

**Comment by Pangchengqiu12** [comment]
要是有docker一键部署就好了
